.gitignore

```diff
@@ -15,4 +15,5 @@ static-root
 /static/stylesheets/style.css
 tmp/*
 media/*
-pydotorg/settings/mylocal.py
\ No newline at end of file
+pydotorg/settings/mylocal.py
+.idea/
```

Comment:
We usually prefer things like this to a global .gitignore. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore for details.

Reply:
Thanks. You never stop learning.